tests/cmake/CMakeLintBearTest.py

```diff
@@ -27,3 +27,11 @@
     invalid_files=(),
     tempfile_kwargs={'suffix': '.cmake'},
     settings={'cmakelint_config': conf_file})
+
+CMakeLintBearConfigTest = verify_local_bear(
+    CMakeLintBear,
+    valid_files=(good_file, bad_file_mixes_case),
+    invalid_files=(),
+    tempfile_kwargs={'suffix': 'cmakelint'},
```

Comment:
Is it `cmakelint` or `.cmake`?

Reply:
I think this is fine. With `.cmake` the tests fails. I have referred https://github.com/richq/cmake-lint  for writing tests.